src/createStreamingManager.ts

```diff
@@ -76,7 +85,11 @@ function pollStats(peerConnection: RTCPeerConnection, onVideoStateChange) {
 
             if (!isStreaming) {
                 onVideoStateChange?.(StreamingState.Start);
-
+                if (streamsCount >= streamsBeforeReady && !getIsConnected()) {
+                    console.log('connecting from video stream')
+                    connect();
```

Comment:
why connect?

Reply:
call is connectionCallback
its more clear when connect that at first i tought is the actul connetion api - but its not
it is just a callback and flag